astropy/io/fits/hdu/compressed/tests/test_tiled_compression.py

```diff
@@ -61,8 +67,12 @@ def test_zblank_support(canonical_data_base_path, tmp_path):
 
     hdu.writeto(tmp_path / "test_zblank.fits")
 
+    with fits.open(tmp_path / "test_zblank.fits", disable_image_compression=True) as hduc:
+        # The ZBLANK value is different from before as we don't preserve it,
+        # instead we always write out with the default ZBLANK
+        assert hduc[1].header['ZBLANK'] == -2147483648
+
     with fits.open(tmp_path / "test_zblank.fits") as hdul:
-        assert "ZBLANK" in hdul[1].header
```

Comment:
@saimn - the one thing that does change is that ZBLANK is no longer in the decompressed header (which is correct)